rules/library.bzl

```diff
@@ -370,6 +370,27 @@ def apple_library(name, library_tools = {}, export_private_headers = True, names
     )
     internal_deps.append(public_hmap_name)
 
+    if swift_sources:
+        generated_swift_header_name = module_name + "-Swift.h"
```

Comment:
The generated header doesn't guarantee to have the suffix "-Swift.h". This can be changed.  I think we should fetch this from providers of `swift_library` rule.

Reply:
This is us setting what the header name will be

Reply:
It looks like another variable is created for this header name further down in the file -- would it be better to reuse the variable declared here?

https://github.com/ob/rules_ios/blob/a690f50025d9c6014f36a309bba669859dc2357f/rules/library.bzl#L500

Reply:
yeah i think we can re-use

Reply:
I find many places also use `"-Swift.h"` directly.